resources/views/user/profile/show.blade.php

```diff
@@ -634,6 +634,14 @@ class="form__button form__button--text"
                     <dd>{{ $user->id }}</dd>
                     <dt>{{ __('common.email') }}</dt>
                     <dd>{{ $user->email }}</dd>
+                    <dt>2FA Enabled</dt>
+                    <dd>
+                        @if ($user->two_factor_confirmed_at != NULL)
```

Comment:
```suggestion
                        @if ($user->two_factor_confirmed_at !== null)
```

PHP generally uses lowercase for null. Also best practice to check for null explicitly instead of falsey values.